### DISCUSSION TITLE
Built-in Screen Record app

### DISCUSSION THREAD

**Discussion by alxmagro** [elementary/triage#423] [Ideas]
I think Ubutu have this resource, nowadays ElementaryOS have only the "Screenshoot" app.
A built-in and integrated "Screen Record" app would be a very useful.

(There is a third one in the store, but it is outdated and it stopped working for me)

**Comment by GuidoBartoli** [comment]
I personally found [Simple Screen Recorder](https://www.maartenbaert.be/simplescreenrecorder/) the most effective solution for Linux, I currently use it on Mint with no issues. I think that would be a good candidate to be included in Elementary OS.

**Reply by vyvir** [reply]
I don't think it would be that good of a solution. As far as I know, Simple Screen Recorder is written in Qt and doesn't run in Wayland.

**Comment by GuidoBartoli** [comment]
> I don't think it would be that good of a solution. As far as I know, Simple Screen Recorder is written in Qt and doesn't run in Wayland.

Ok, I didn’t notice that, thanks.
